### PR TITLE
Update QUEST_LV_0300.tsv

### PR DESCRIPTION
**Pull request by Satoram**
Some editing of these lines. These lines will likely need to be revised again in order to make sure everything makes sense in context.
There were some lines that I was unable to make sense of.

### CODE REVIEW

**Comment by Satoram** [comment]
Thank you, I have gone over the range of lines again and made some changes, based on what you said as well as what I missed going over it the first time.

File: QUEST_LV_0300.tsv [hunk 1]
```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
...
+QUEST_LV_0300_20151001_000268	The last barrier stone must be exploding by now.{nl}
 QUEST_LV_0300_20151001_000269	As the monsters were being petrified, their last breathes were also solidified... Throw those against barrier stones.{nl}They will gradually collapse.
-QUEST_LV_0300_20151001_000270	The core of the barrier stone is composed of the condensed spirits.{nl}We will make orbs by collecting those that would explode the integrator.
-QUEST_LV_0300_20151001_000271	I will put my last power onto the cores of barrier stones which you've collected.{nl}Pull out monsters' detonators with this and make an orb.{nl}
-QUEST_LV_0300_20151001_000272	Of course, I will give you a method to make the orb.{nl}The orb that would explode the integrator.{nl}
+QUEST_LV_0300_20151001_000270	The core of the barrier stone is composed of condensed spirits.{nl}We will make orbs by collecting those that would destroy the integrator.
+QUEST_LV_0300_20151001_000271	I will put my last power into the cores of barrier stones which you've collected.{nl}Pull out monsters' detonators with this and make an orb.{nl}
```
Comment (Satoram) [inline QUEST_LV_0300.tsv]: I am not sure what the last sentence of this line means.

**Comment by sususuyo** [comment]
No problem! I haven't been able to look through the whole PR yet so I'll add more notes if I see anything off. Good work so far! :+1:

File: QUEST_LV_0300.tsv [hunk 2]
```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
...
-QUEST_LV_0300_20151001_000239	I've tried my best to escape for hundreds of years continuously.{nl}But, the curse of the petrification is the power of the Demon Queen. {nl}The endless depression is suffocating me.{nl}
-QUEST_LV_0300_20151001_000240	Even if you escape from here, you are just betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}
-QUEST_LV_0300_20151001_000241	If you cooperate on my death, I will definitely remove your curse.{nl}I promise in the name of Giltine.
-QUEST_LV_0300_20151001_000242	It doesn't matter what happens to my body.{nl}I just want to take some rest...
+QUEST_LV_0300_20151001_000239	I've tried my best to escape for hundreds of years.{nl}But, the curse of the petrification is the power of the Demon Queen. {nl}This endless depression is suffocating me.{nl}
+QUEST_LV_0300_20151001_000240	Even if you escape from here, you are just a betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}
```
Comment (sususuyo) [inline QUEST_LV_0300.tsv]: "Even if you escape from here, you are just a betrayer. You will get the word "betrayer". {nl}While going to the feet of the colleagues who mocked you...{nl}"

> Even if you escape, you're still a traitor.{nl}You'll be called a "traitor" while at the feet of the colleagues who mocked you...{nl}

File: QUEST_LV_0300.tsv [hunk 3]
```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
...
+QUEST_LV_0300_20151001_000263	Well done. My death is near.
+QUEST_LV_0300_20151001_000264	The next barrier stone is up there. {nl}We have to put in great power all at once. Like an explosion.{nl}
+QUEST_LV_0300_20151001_000265	Okay.. there was a grave robber among the ones who were sacrificed.{nl}I tried to lead myself into death using a bomb, but it was useless.
 QUEST_LV_0300_20151001_000266	We should not forget about the core of the barrier stone.{nl}To avoid being cursed again, I will cast a spell again.
-QUEST_LV_0300_20151001_000267	How come I never thought about this.{nl}Maybe my loyalty has surpassed the fear of lonliness.
-QUEST_LV_0300_20151001_000268	The last barrier stone at the deep corner must be exploding by now.{nl}
+QUEST_LV_0300_20151001_000267	How come I never thought about this.{nl}Maybe my loyalty has surpassed the fear of loneliness.
+QUEST_LV_0300_20151001_000268	The last barrier stone must be exploding by now.{nl}
 QUEST_LV_0300_20151001_000269	As the monsters were being petrified, their last breathes were also solidified... Throw those against barrier stones.{nl}They will gradually collapse.
```
Comment (sususuyo) [inline QUEST_LV_0300.tsv]: I know you didn't touch this line but can you change "breathes" to "breaths"?

File: QUEST_LV_0300.tsv [hunk 4]
```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
...
 QUEST_LV_0300_20151001_000289	(It seems that some letters are visible, but I can't clearly see them.)
 QUEST_LV_0300_20151001_000290	To be able to read the letters, you must have a piece of the symbol.
 QUEST_LV_0300_20151001_000291	{nl}You can obtain the piece of the symbol somewhere in Galeed Plateau.
-QUEST_LV_0300_20151001_000292	(Something is written, but is not readable.)
+QUEST_LV_0300_20151001_000292	(Something is written, but is not legible.)
 QUEST_LV_0300_20151001_000293	(It seems that something is written on it, but it can't be read in this condition.)
-QUEST_LV_0300_20151001_000294	That ring was given by my mother.{nl}I want to go back to home where my mother is with this ring...
-QUEST_LV_0300_20151001_000295	Our house is in the inner castle district...{nl}Please... I want to go back to home. With the ring...
-QUEST_LV_0300_20151001_000296	Yes. Gailus Legwyn is my old friend...{nl}But even an old friend shouldn't just take a precious object and not return.{nl}
+QUEST_LV_0300_20151001_000294	That ring was given to me by my mother.{nl}I want to go back to home where my mother is with this ring...
```
Comment (sususuyo) [inline QUEST_LV_0300.tsv]: "I want to go back to home where my mother is with this ring..."

> I want to go back home with it, back to where my mother is...

File: QUEST_LV_0300.tsv [hunk 5]
```diff
@@ -209,91 +209,91 @@ QUEST_LV_0300_20150908_000208	You've failed to collect pertification powder!
...
+QUEST_LV_0300_20151001_000295	Our house is in the inner castle district...{nl}Please... I want to go back home. With the ring...
```
Comment (sususuyo) [inline QUEST_LV_0300.tsv]: "I want to go back home. With the ring..."

> I want to go back home with the ring...

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**